app/src/main/java/com/sci/recipeandroid/feature/auth/data/datasource/remote/AuthRemoteDataSourceImpl.kt

```diff
@@ -0,0 +1,19 @@
+package com.sci.recipeandroid.feature.auth.data.datasource.remote
+
+class AuthRemoteDataSourceImpl(): AuthRemoteDataSource {
+    override suspend fun googleAuthentication(token:String): Result<String> {
```

Comment:
authenticateWithGoogle
authenticateWithFacebook

function name should be verb

Reply:
change the name of this function 9baf80468f57d78d6c4b13f545e1ebaa10f9d59c